app/mailers/event/participant_reminder_mailer.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class Event::ParticipantReminderMailer < ApplicationMailer
+  include EventMailer
+  include MultilingualMailer
+
+  REMINDER = "event_participant_reminder"
+
+  def reminder(participation)
+    @participation = participation
```

Comment:
Hier ist dann etwas versteckt das es das at participations im modul eventmailer braucht. instanzvariablen in einem modul zu brauchen ist nicht so optimal. Ich finde das in diesem fall, aber okay, weil notwendig.